packages/wp-source/src/actions.ts

```diff
@@ -265,5 +273,9 @@ const actions: WpSource["actions"]["source"] = {
       });
     }
   },
+
+  afterSSR: ({ state }) => {
+    delete state.source.auth;
```

Comment:
The `frontity.options.sourceAuth` prop is not been removed here and thus it would be present in the state. I think it is not a big deal because the auth token would be already in the URL anyway. Just so you know.

Reply:
Good catch David. I think we should remove it as well.

Reply:
Solved 👍